src/Generators/AbstractTagGenerator.php

```diff
@@ -80,123 +112,115 @@ public function getTagsAsString()
     {
         $tagString = '';
 
-        foreach($this->tags as $tag) {
+        foreach ($this->tags as $tag) {
             $tagString .= ' * ' . $tag . "\n";
         }
 
         return $tagString;
     }
 
     /**
-     * List of supported tags.
-     *
-     * Each tag type can hold many tags, so we keep them grouped.
-     * Also used to reset the tag list after each run
-     *
-     * @return array
-     */
-    public function getSupportedTagTypes()
-    {
-        return array(
-            'properties'=> array(),
-            'methods'   => array(),
-            'mixins'    => array(),
-            'other'     => array()
-        );
-    }
-
-    /**
-     * @return void
-     */
-    abstract protected function generateTags();
-
-    /**
      * Generate the mixins for DataExtensions.
      */
     protected function generateExtensionsTags()
     {
         if ($fields = (array)$this->getClassConfig('extensions')) {
             foreach ($fields as $fieldName) {
-                $this->pushMixinTag($fieldName);
+                $this->pushMixinTag("\\$fieldName");
             }
         }
     }
 
     /**
-     * Generate the Owner-properties for extensions.
+     * @param $key
+     * @return mixed
      */
-    protected function generateOwnerTags()
+    protected function getClassConfig($key)
     {
-        $className = $this->className;
-        $owners = (array)array_filter($this->extensionClasses, function($class) use ($className) {
-            $config = Config::inst()->get($class, 'extensions', Config::UNINHERITED);
-            return ($config !== null && in_array($className, $config, null));
-        });
-
-        if (!empty($owners)) {
-            if($this->reflector->isSubclassOf('DataExtension')) {
-                $owners[] = $this->className;
-            }
-            $this->pushPropertyTag(implode("|", $owners) . " \$owner");
-        }
+        return Config::inst()->get($this->className, $key, CONFIG::UNINHERITED);
```

Comment:
This should be `Config::UNINHERITED` just in case

Reply:
Yeps, good catch